src/app/static/src/app/components/baseline.vue

```diff
@@ -8,201 +8,26 @@
 <script lang="ts">
     import Vue from "vue";
     import {DynamicForm, DynamicFormMeta} from "@reside-ic/vue-dynamic-form";
+    import {Project, Region} from "../models/project";
+    import {mapState} from "vuex";
 
-    interface Data{
+    interface Computed {
+        currentProject: Project,
         options: DynamicFormMeta
     }
 
-    export default Vue.extend<Data, {}, {}, {}>({
+    export default Vue.extend<{}, {}, Computed, {}>({
         components: {DynamicForm},
-        data(): Data {
-            return {
-                options: baselineOptions
-            }
+        computed: {
+            ...mapState(["currentProject"]),
+            options:  {
+                get() {
+                    return this.currentProject.currentRegion.baselineOptions
+                },
+                set(value: DynamicFormMeta) {
+                      //TODO: update by calling a mutation to set options in the current region
```

Comment:
Code cov seems to be complaining about this, which will be implemented and tested in the next ticket..